SofaKernel/modules/SofaEigen2Solver/CMakeLists.txt

```diff
@@ -1,23 +1,23 @@
 cmake_minimum_required(VERSION 3.12)
 project(SofaEigen2Solver LANGUAGES CXX)
 
+# message(WARNING "SofaEigen2Solver has been removed, the package still exists but is empty. Eigen classes have moved to Sofa.LinearAlgebra and SVDLinearSolver to SofaDenseSolver.")
```

Comment:
Nope, it is just to avoid potentially a lot of warnings messages for the moment.
It must be uncommented before the next release (while doing the deprecation sweep)